### ISSUE TITLE
Enable alt text for images

### ISSUE THREAD

**Issue by benerachel** [benerachel/d4i#8]
As a user with visual impairment I want to enjoy the added value of any images included in an app or webform. 

An 'alt text' should be present or, set 'alt=null' if the image is just for decorative reasons.
